### ISSUE TITLE
Organization of the blog posts

### ISSUE THREAD

**Issue by omoindrot** [cs230-stanford/cs230-code-examples#7]
### General (common between TensorFlow and PyTorch)
1. Introduction to project starter code
2. Logging + hyperparams
3. AWS setup
4. Train/Dev/Test set

### TensorFlow
1. Getting started
2. Dataset pipeline: `tf.data`
3. Creating the model (`tf.layers`) + training + evaluation
  - model
  - training ops
  - input_fn and model_fn
  - evaluation and `tf.metrics`
  - initialization
  - saving
  - tensorboard
  - global_step
